### DISCUSSION TITLE
Question on target_vocabulary_id B2AI-DelphiR2 from Bridge2AI Delphi spreadsheet with respect to overlap of target_domain_id 'Measurement' vs 'Measurement/Obs'

### DISCUSSION THREAD

**Discussion by heschmidt04** [chorus-ai/StandardsModule#85] [Flowsheet Questions] [answered]
I'm working off the [Bridge2AI stcm delphi spreadsheet] that I loaded into our local database. (https://docs.google.com/spreadsheets/d/1gTCCOcf6b8yoVtRin711Mn0ZD5pd9iCJ4EDAss4yJao/edit?usp=sharing)

@jshoughtaling and folks -- 
What OMOP target table is the target_domain_id of 'Measurement/Obs' being used to provide the athena concept id? Is that supposed to be [Observation](https://ohdsi.github.io/CommonDataModel/cdm54.html#OBSERVATION)? 

I notice that I have an overlap of 'Measurement' values in 'Measurement/Obs'. 
The entries that are only in 'Measurement/Obs' are target_vocabulary_id B2AI-DelphiR2 and have concept ids assigned in the 2 billion id range. 

The OMOP CDM 5.4 documentation says the following for OMOP Observation table: 

> ETL Conventions
> 
> Records whose Source Values map to any domain besides Condition, Procedure, Drug, Measurement or Device should be stored in the Observation table. Observations can be stored as attribute value pairs, with the attribute as the Observation Concept and the value representing the clinical fact. 
> 

What do I do in the scenario if I have an Observation that matches with 'Measurement/Obs' but is also in 'Measurement'?
Can I *use* the loinc or snomed vocabulary concept id instead of the target_vocabulary_id B2AI-DelphiR2 2billion range target_concept_id? 

Are the target_concept_ids for OMOP Measurement table vs Observation table supposed to be different or can they overlap as I am asking in the previous question? 

Thanks
Heidi

**Comment by jshoughtaling** [comment]
@heschmidt04 - great questions. I believe @p-talapova should be able to provide some insight into why those Measurement/Obs hybrid domain values were used and how to interpret their meaning in a broader context. As a quick answer to your second question, I would say any standard (LOINC/SNOMED) concepts tied to your source flowsheet description should supersede any 2B+ concepts specific to the B2AI-Delphi process.

**Comment by p-talapova** [comment, marked as answer]
Hi there, 

thank you, @jshoughtaling!

> What OMOP target table is the target_domain_id of 'Measurement/Obs' being used to provide the athena concept id? Is that supposed to be Observation?

@heschmidt04, we advise caution with the extensive use of the B2AI 2-billion terms, as their development is ongoing. Should your work necessitate these terms, you may choose to utilize either the Observation or Measurement table for each specific term. However, it is important to ensure that a single standard target_concept_id is not simultaneously allocated to both tables.

In general, the [Measurement/Obs](https://athena.ohdsi.org/search-terms/terms/44) domain is one of the predefined domains within the OMOP standard, signifying concepts that could be allocated to either the [Observation](https://ohdsi.github.io/CommonDataModel/cdm54.html#OBSERVATION) or [Measurement ](https://ohdsi.github.io/CommonDataModel/cdm54.html#MEASUREMENT) tables depending on the context of the data, particularly the units and values associated with each entry.

Regarding the source-to-concept map table with Delphi mappings, we currently employ this domain designation as a temporary solution for source terms that are being considered for inclusion in the OMOP vocabulary. Evaluating the appropriate domain for each potential concept requires additional effort, and we have plans to undertake this task and provide updates in the near future.

> What do I do in the scenario if I have an Observation that matches with 'Measurement/Obs' but is also in 'Measurement'?
Can I use the loinc or snomed vocabulary concept id instead of the target_vocabulary_id B2AI-DelphiR2 2billion range target_concept_id?

This scenario is problematic, and we encourage reporting it to our team because it can highlight potential standard duplicates. However, if you have an observation that matches with 'Measurement/Obs' but is also with 'Measurement', you should opt for 'Measurement'. 
If you aim to improve mapping, it is acceptable to utilize LOINC, SNOMED, or any other standard concepts from various OMOP vocabularies instead.

> Are the target_concept_ids for OMOP Measurement table vs Observation table supposed to be different or can they overlap as I am asking in the previous question?

In the OMOP model, target_concept_ids for the Measurement and Observation tables are ideally expected to be distinct, but there can be instances of overlap in existing OMOP vocabularies. As mentioned, such overlaps (exact semantic replicates) should be carefully considered and potentially reported.

Heidi, thank you so much for your proactive efforts and enthusiasm. 
Your involvement is incredibly valuable to us!
